go/teams/service_helper.go

```diff
@@ -991,9 +991,15 @@ func IgnoreRequest(ctx context.Context, g *libkb.GlobalContext, teamname, userna
 		return err
 	}
 	arg := apiArg(ctx, "team/deny_access")
-	arg.Args.Add("team", libkb.S{Val: teamname})
+	arg.Args.Add("team", libkb.S{Val: teamName})
 	arg.Args.Add("uid", libkb.S{Val: uv.Uid.String()})
-	_, err = g.API.Post(arg)
+	if _, err := g.API.Post(arg); err != nil {
+		t, err := GetForTeamManagementByStringName(ctx, g, teamName, true)
```

Comment:
Is there a better way to do this?

Reply:
this looks good but i'd flatten the flow like:
```
err = api.post
if err != nil { return err }
t, err = GetTeamForManagementByStringName
if err != nil { return err }
... etc
```

Reply:
ah yes this is better